src/util/mod.rs

```diff
@@ -11,6 +11,8 @@ pub mod address;
 /// Allocators
 // This module is made public so the binding could implement allocator slowpaths if they would like to.
 pub mod alloc;
+/// Helpers for making native APIs.
+pub mod apiutils;
```

Comment:
Could use the name `api_util` to be consistent with other existing modules such as `rust_util`/`test_util`.